cfgov/jinja2/v1/newsroom/press-resources/index.html

```diff
@@ -160,7 +160,7 @@ <h2 class="press-section_title press-photos-bios_title">Photos and bios</h2>
             <section class="contact-person contact-person__director content-l_col-1-2">
                 <div class="media">
                     <img class="media_image media_image__150 contact-person_photo"
-                         src="{{ static('img/director-cordray-round-300x300.jpg')}"
+                         src="{{ static('img/director-cordray-round-300x300.jpg')}}"
```

Comment:
Trivial: A space before the closing `{{` here would make this typo more obvious in the future :)